### ISSUE TITLE
Structure searching does not use random seeds

### ISSUE THREAD

**Issue by woutheijnen** [Zodsmar/SeedSearcherStandaloneTool#62]
Hello,
I tried to make a pull request as I fixed this issue I was having but it is not letting me (unauthorized) so I contact you through a github issue.

When you open the SASSA, leave the default options and pick one structure like for example the monument and then press search, the next time you press search again you will get exactly the same list of seeds as the seeds are not randomly picked.
This also persists if you search, close the app, choose the same structure and search again.

**Comment by woutheijnen** [comment]
The way I fixed this was as follow in Searcher.java in searchRandomly method : 

```
public static void searchRandomly(int searchSize, long startSeedStructure, long endSeedStructure, Collection<StructureProvider> sList, Collection<StructureProvider> soList, Collection<Biome> bList, Collection<Biome> boList, Collection<Biome.Category> cList, Collection<Biome.Category> coList, Dimension dimension, int incrementer, int biomePrecision, MCVersion mcVersion) {
        Vec3i origin = Vec3i.ZERO;
        ThreadLocalRandom threadLocalRandomizer = ThreadLocalRandom.current();
        ChunkRand rand = new ChunkRand();
        int totalStructures = sList.size();
        Map<StructureProvider, List<CPos>> structures = new HashMap<>();
        sList = sList.stream().distinct().collect(Collectors.toList());
        for(long structureSeedIncrementer = startSeedStructure; structureSeedIncrementer < endSeedStructure; structureSeedIncrementer++, structures.clear()) {
            long structureSeed = threadLocalRandomizer.nextLong(startSeedStructure, endSeedStructure);
            for(StructureProvider searchProvider: sList) {
                ...
```

And to get `endSeedStructure` in fxmlController.java at line 303 : 
```
long endStructureSeed = Math.min((long) Math.floor(Math.pow(2, 48)/singleton.getAmountOfCores().getValue() * (i+1)), 1L << 48);
```

**Comment by Zodsmar** [comment]
This is one way of fixing it but there is also a Biome Precision option that I haven't implemented which should also fix this.

**Comment by Zodsmar** [comment]
Fixed and I credited you in the commit f623fe0
